### ISSUE TITLE
When the file is opened in the workspace during remote development, and the window is reloaded, the window layout is abnormal.

### ISSUE THREAD

**Issue by czfadmin** [microsoft/vscode#119049]
- VS Code Version: 1.55.0-insider
- OS Version:  Linux x64 3.10.0-1127.19.1.el7.x86_64

When using centos for remote development, the layout will be abnormal, as shown below.
![image](https://user-images.githubusercontent.com/13393959/111266917-2c1fcd80-8666-11eb-9f0f-c263443fe35e.png)

Steps to Reproduce:
1. When I previously connected to the remote server, a file was already opened in the workspace during the previous connection. After I reloaded Windows, I did not enter the password to verify.
2. At this time, the code appears in a blank environment, as shown below
![image](https://user-images.githubusercontent.com/13393959/111267064-66896a80-8666-11eb-9640-c12e1ea491d1.png)
3. Use shortcut key `Ctrl+ Shift + P`
![image](https://user-images.githubusercontent.com/13393959/111267179-933d8200-8666-11eb-9983-4f847c716a2d.png)
4. After entering the password to verify the remote connection, the layout returns to normal.
5. When I reloaded the code after not opening the software in the previous connection, the above layout abnormality did not appear.

Does this issue occur when all extensions are disabled?: Yes

When you want to verify whether the extension is causing the problem in the remote development environment, select the command to disable the extension and reload the window in the command panel, and hope that the disabling of the plug-in related to remote development can be excluded, because the above The command to disable extensions and reload the window also disables the remotely developed plugins, so you need to re-enable the relevant plugins and reload the window, so that other plugins that you don't want to close are also enabled.

**Comment by dbaeumer** [comment]
@sbatten starting with you since this should normally not be affected by remote development.

**Comment by sbatten** [comment]
@czfadmin I am having a bit of trouble following the repro steps so I'm sorry about that. I'll try to restate and please correct any steps.

1. Connect from windows to a linux host for ssh development
2. The layout is blank
3. Use the command palette to reload the window
4. Enter the password for the ssh remote
5. Layout is normal?

Can you clarify what steps are needed to make sure 1 is different from 4?

Also, can you describe the abnormality? It appears to me that in the first shot, the layout is half blank on top of a normal window. Is that a correct assessment? A single window split into 2?

**Comment by czfadmin** [comment]
@sbatten First of all, I used to connect to the remote server before.At this time, I have opened a file in the work area, and then reloaded the code, and the layout will be disordered.I call the shortcut key to display the terminal and enter the password to verify.After the verification, the layout returns to normal. When the file reload window is not opened during the first remote connection, the above layout exception problem will not occur.
The difference between step 1 and step 4 you described is that first you connect to the remote server for the first time and ***open a file***, and then reload the window, and you will find this live line at this time. Only enter the password verification in the four steps, and the layout will return to normal at this time.
I did not split the working range in the same window into two
